openchs-server-api/src/main/java/org/openchs/service/IndividualService.java

```diff
@@ -143,23 +145,37 @@ public IndividualContract getSubjectInfo(String individualUuid) {
     }
 
     public List<RelationshipContract> constructRelationships(Individual individual) {
-        return individual.getRelationships().stream().map(individualRelationship -> {
-            RelationshipContract relationshipContract = new RelationshipContract();
-            relationshipContract.setUuid(individualRelationship.getUuid());
-            relationshipContract.setIndividualBUuid(individualRelationship.getIndividualB().getUuid());
-            relationshipContract.setIndividualBIsToARelation(individualRelationship.getRelationship().getIndividualBIsToA().getName());
-            relationshipContract.setRelationshipTypeUuid(individualRelationship.getRelationship().getUuid());
-            relationshipContract.setEnterDateTime(individualRelationship.getEnterDateTime());
-            relationshipContract.setExitDateTime(individualRelationship.getExitDateTime());
-            relationshipContract.setFirstName(individualRelationship.getIndividualB().getFirstName());
-            relationshipContract.setLastName(individualRelationship.getIndividualB().getLastName());
-            relationshipContract.setDateOfBirth(individualRelationship.getIndividualB().getDateOfBirth());
-            relationshipContract.setVoided(individualRelationship.isVoided());
-            if (individualRelationship.getExitObservations() != null) {
-                relationshipContract.setExitObservations(constructObservations(individualRelationship.getExitObservations()));
-            }
-            return relationshipContract;
+        List<RelationshipContract> relationshipContractList1 = individual.getRelationshipsWithB().stream().filter(individualRelationship -> !individualRelationship.isVoided()).map(individualRelationship -> {
```

Comment:
@amolnirmalpsl How about we rename relationshipContractList1 to relationshipsFromSelfToOthers and relationshipContractList2 to relationshipsFromOthersToSelf. Probably the same for fields on Individual also?